cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -32,7 +32,8 @@ spec:
             - --sink=gcm
             - --sink=gcmautoscaling
             - --sink=gcl
-            - --sink_frequency=2m
+            - --stats_resolution=30s
+            - --sink_frequency=1m
```

Comment:
Previously (before #14210) stats_resolution was 1m to not load GCM too much. Also cluster size autoscaling to work correctly requires the resolution to be at least 15s (or decreasing [maxHousekeepingInterval](https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/cadvisor/cadvisor_linux.go#L47) in Kubelet).